docs/function-mesh-worker/function-mesh-worker-overview.md

```diff
@@ -28,4 +56,4 @@ This table lists the version mapping relationships between Function Mesh and Fun
 | v0.2.0| <br />- v2.10.0.5-v2.10.0.6 <br />- v2.9.2.18-v2.9.2.22 <br />- v2.8.3.4 |
 | v0.3.0| <br />- v2.10.0.7 <br />- v2.9.2.23+ <br />- v2.8.3.5+ |
 | v0.4.0| <br />- v2.10.1.4+ <br />- v2.9.3.3+ <br />- v2.8.3.6+ |
-| v0.5.0| <br />- v2.10.1.7+ <br />- v2.9.3.5+ <br />- v2.8.4.1+ |
\ No newline at end of file
+| v0.5.0| <br />- v2.10.1.7+ <br />- v2.9.3.5+ <br />- v2.8.4.1+ |
```

Comment:
Confirmed with Rui, For FM v0.5, the FM worker service versions should be the following. Could you please also help update it? Thanks.
Function Mesh 0.5.0:
v2.10.1.7 ～ 2.10.1.8
v2.9.3.6 ～2.9.3.8
v2.8.4.1

Reply:
ok